src/NzbDrone.Core/Indexers/IIndexerSettings.cs

```diff
@@ -5,5 +5,7 @@ namespace NzbDrone.Core.Indexers
     public interface IIndexerSettings : IProviderConfig
     {
         string BaseUrl { get; set; }
+
+        int SearchPriority { get; set; }
```

Comment:
This should be part of the definition, not part of settings, same as download clients. It should also just be called priority, since it doesn't strictly apply to searching.

Reply:
Didnt even know DC had a priority, will have a look and change accordingly